Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1752,6 +1752,12 @@ class BrowserViewController: UIViewController,
             case .formSubmit:
                 self?.showCreditCardAutofillSheet(fieldValues: fieldValues)
                 break
+            case .fillAddressForm:
+                // TODO: FXIOS-7670
+                return
+            case .captureAddressForm:
+                // TODO: FXIOS-7670
+                return
```

Comment:
What about credit cards? 

We need to figure out when a user taps on a address field vs credit card field so that in the accessory view er can change it and show the correct view there `tabWebView.accessoryView.reloadViewFor(.creditCard)`